tests/unit_tests/c_wrappers/res/enkf/test_res_config.py

```diff
@@ -259,24 +259,17 @@ def test_extensive_config(setup_case):
         assert exp_job_data["STDERR"] == job_list[job_name].get_stderr_file()
         assert exp_job_data["STDOUT"] == job_list[job_name].get_stdout_file()
 
-    ecl_config = res_config.ecl_config
+    ensemble_config = res_config.ensemble_config
     for extension in ["SMSPEC", "UNSMRY"]:
         assert (
             Path(snake_oil_structure_config["REFCASE"] + "." + extension).resolve()
-            == Path(ecl_config.refcase.case + "." + extension).resolve()
+            == Path(ensemble_config.refcase.case + "." + extension).resolve()
         )
     assert (
         Path(snake_oil_structure_config["GRID"]).resolve()
-        == Path(ecl_config._grid_file).resolve()
+        == Path(ensemble_config._grid_file).resolve()
     )
 
-    ensemble_config = res_config.ensemble_config
-    assert set(
-        snake_oil_structure_config["SUMMARY"]
-        + snake_oil_structure_config["GEN_KW"]
-        + snake_oil_structure_config["GEN_DATA"]
-    ) == set(ensemble_config.alloc_keylist())
-
```

Comment:
how come this block was removed? sorry, it's not immediately clear to me

Reply:
I thought this was introduced when rebasing on main. After some investigation, i see this is not the case.
I've reverted this snippet since this is not my intended change.